### PR TITLE
Added share the app functionality in the about fragment

### PR DESCRIPTION
**Pull request by Devansh-gaur-1611**

### CODE REVIEW

**Comment by VishnuSanal** [comment]
@Devansh-gaur-1611 Please run spotless -> Ctrl Ctrl `./gradelw/spotlessApply`

**Comment by VishnuSanal** [comment]
@Devansh-gaur-1611 Are you on Android Studio? If yes, press Ctrl twice - a dialog (run anything) pops up. Type in
./gradlew spotlessApply
which would run spotless task. This is used to ensure formatting rules. Then, some files would've changed. Commit 'em & push :)

**Comment by Devansh-gaur-1611** [comment]
@VishnuSanal, Thanks for your valuable advice, it has worked, thanks a lot

File: .idea/misc.xml [hunk 1]
```diff
@@ -18,6 +18,8 @@
         <entry key="..\:/AndroidStudioProjects/Quotes/app/src/main/res/layout/share_action_option_pick_dialog.xml" value="0.1" />
         <entry key="..\:/AndroidStudioProjects/Quotes/app/src/main/res/layout/share_layout.xml" value="0.1" />
         <entry key="..\:/AndroidStudioProjects/Quotes/app/src/main/res/layout/tour_single_item.xml" value="0.1" />
+        <entry key="..\:/Users/hp/Desktop/Hacktober Fest/Quotes/app/src/main/res/drawable/ic_done.xml" value="0.2203125" />
+        <entry key="..\:/Users/hp/Desktop/Hacktober Fest/Quotes/app/src/main/res/layout/fragment_about.xml" value="0.20416666666666666" />
```
Comment (VishnuSanal) [inline .idea/misc.xml:22]: Please revert

File: app/src/main/java/phone/vishnu/quotes/fragment/AboutFragment.java [hunk 2]
```diff
@@ -107,8 +107,17 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                         }
 
                     } else if (position == 4) {
-                        openLink("https://github.com/VishnuSanal/Quotes");
+                        Intent intent = new Intent(Intent.ACTION_SEND);
+                        intent.setType("text/plain");
+                        intent.putExtra(
+                                Intent.EXTRA_TEXT,
+                                "Install Quotes Status Creator - an open source app that lets you share quotations as status images on social media\n\nhttps://play.google.com/store/apps/details?id=phone.vishnu.quotes\n");
```
Comment (VishnuSanal) [inline app/src/main/java/phone/vishnu/quotes/fragment/AboutFragment.java:114]: Remove the `\n` at the end

**Review by VishnuSanal — CHANGES_REQUESTED** [review]
@Devansh-gaur-1611 

Looks good to me, other than these couple things. Please see the comments & address them

File: .idea/misc.xml (hunk 1 above)
Comment (Devansh-gaur-1611) [reply .idea/misc.xml:22]: Ok, but I have a doubt. Actually, there is a change in idea/misc.xml apart from these 2 changes so if I revert then that change will also get reverted, so I want to ask that should I just delete these two lines or is it necessary to revert the code?

Comment (VishnuSanal) [reply .idea/misc.xml:22]: > Ok, but I have a doubt. Actually, there is a change in idea/misc.xml apart from these 2 changes so if I revert then that change will also get reverted, so I want to ask that should I just delete these two lines or is it necessary to revert the code?

@Devansh-gaur-1611 Um.. The contents of the file should be exactly same of what's in the source repo (i.e, this repo). If this is so, it wont show up in differences.

**Review by VishnuSanal — APPROVED** [review]

**Comment by VishnuSanal** [comment]
@Devansh-gaur-1611 Thanks for your work! :)

**Comment by Devansh-gaur-1611** [comment]
> @Devansh-gaur-1611 Thanks for your work! :)

Its my pleasure and also thanks for helping me in solving issue at various moments

**Pull request merged**